apps/avatax/src/lib/app-configuration-logger.test.ts

```diff
@@ -81,13 +81,13 @@ describe("AppConfigurationLogger", () => {
     );
 
     expect(mockInfo).toHaveBeenCalledWith("Received configuration", {
-      address: JSON.stringify({
+      address: {
         city: "test",
         country: "test",
         zip: "10111",
         state: "NY",
         street: "test",
-      }),
```

Comment:
interesting that it works but technically js doesn't preserve keys order (or guarantees it), so it may be flaky - one day stringified object can be different

but non blocking, if it works, we dont have to touch it